src/domains/ioweb-common/06_cdn_itn.tf

```diff
@@ -47,3 +52,159 @@ module "io_web_profile_itn_fe_st" {
 
   tags = var.tags
 }
+
+resource "azurerm_cdn_frontdoor_profile" "portal_profile" {
+  name                = format("%s-profile-portal-afd-01", local.project_itn)
+  resource_group_name = azurerm_resource_group.io_web_profile_itn_fe_rg.name
+  sku_name            = "Standard_AzureFrontDoor"
+
+  tags = var.tags
+}
+
+data "azurerm_key_vault_certificate" "portal_custom_certificate" {
+  name         = "account-ioapp-it"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_cdn_frontdoor_secret" "portal_certificate" {
+  name                     = "certificate"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  secret {
+    customer_certificate {
+      key_vault_certificate_id = data.azurerm_key_vault_certificate.portal_custom_certificate.id
+    }
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "portal_custom_domain" {
+  name                     = format("%s-profile-fdd-01", local.project_itn)
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  dns_zone_id = data.azurerm_dns_zone.ioapp_it.id
+  host_name   = "account.ioapp.it"
+
+  tls {
+    # Certificate managed by us and put in a kv
+    certificate_type        = "CustomerCertificate"
```

Comment:
I did some tests because it's the first time I use this CDN service (the one used by the wallet team is actually different), and I realized that it is still possible to use custom domains with managed certificates.
Then, I suggest to use `ManagedCertificate` as value here, and remove all the references to key vault. This means we don't need the pipeline neither. Having a full managed certificate is definitely a better approach. On the other hand, I suppose we need to add `azurerm_dns_txt_record` and `azurerm_dns_cname_record` to the chosen DNS zone as described in [tf doc](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_custom_domain)

I haven't click on `Create` on my tests, so there's still the possibility that something goes wrong and we need to restore the reference to the KeyVault.. But I don't think we will